run.js

```diff
@@ -11,68 +12,120 @@ const pathFor = mode => path.join(
   'react-dom.' + mode + '.js'
 );
 
+// set up AST files.
 const ReactDOMProd = pathFor('production.min');
 const ReactDOMDev = pathFor('development');
 
-console.time('parse prod');
 const prod = fs.readFileSync(ReactDOMProd, 'utf8');
 const prodAst = babylon.parse(prod);
-console.timeEnd('parse prod');
+fs.writeFileSync('react-dom.production.min.ast', 'module.exports = ' + JSON.stringify(prodAst) + ';');
+fs.writeFileSync('react-dom.production.min.js', 'module.exports = ' + JSON.stringify(prodAst) + ';');
+fs.writeFileSync('react-dom.production.min.json', JSON.stringify(prodAst));
 
-console.time('parse dev');
 const dev = fs.readFileSync(ReactDOMDev, 'utf8');
 const devAst = babylon.parse(dev);
-console.timeEnd('parse dev');
-console.log('');
+fs.writeFileSync('react-dom.development.ast', 'module.exports = ' + JSON.stringify(devAst) + ';');
+fs.writeFileSync('react-dom.development.js', 'module.exports = ' + JSON.stringify(devAst) + ';');
```

Comment:
are the require `.js` and `.ast` tests now the exact same? (previously the `.ast` file was the manual read+`JSON.parse` test)`